code/web/src/modules/user/EditProfile.js

```diff
@@ -8,51 +14,59 @@ class EditProfile extends Component {
         super()
         this.state = {
             //image: ?
```

Comment:
Just added this attribute to the User model. Once we figure out the bug in my PR, image will be available to edit. It'll come with a default image URL as well